lib/livebook_web/live/session_live/render.ex

```diff
@@ -1413,4 +1434,12 @@ defmodule LivebookWeb.SessionLive.Render do
   defp starred?(file, starred_files) do
     Enum.any?(starred_files, &Livebook.FileSystem.File.equal?(&1, file))
   end
+
+  def slice(text, min..max_length//_) do
```

Comment:
I think you can do this with CSS with the text-overflow property. Which is better to do in CSS anyway, because people can use the browser zoom, which always changes how much text would fit in the sidebar.